hcl2template/types.packer_config.go

```diff
@@ -367,44 +362,37 @@ func (cfg *PackerConfig) GetBuilds(opts packer.GetBuildsOptions) ([]packer.Build
 			// validate user input against what will become available. Otherwise,
 			// only pass the default variables, using the basic placeholder data.
 			generatedPlaceholderMap := packer.BasicPlaceholderData()
-			if generatedVars != nil {
-				for _, k := range generatedVars {
-					generatedPlaceholderMap[k] = ""
-				}
+			unknownBuildValues := map[string]cty.Value{}
+			for _, k := range generatedVars {
+				generatedPlaceholderMap[k] = fmt.Sprintf("Build_%s. "+
```

Comment:
The `generatedPlaceholderMap` can be removed, no?  I can't find where it's being used.

Reply:
Ah right !! I'm created the `BuilderDataCommonKeys` for this.